### ISSUE TITLE
SCANNING_COMPLETED webhook http/slack payload missing tag

### ISSUE THREAD

**Issue by dkulchinsky** [goharbor/harbor#14293]
**Expected behavior and actual behavior:**
Configure an http and/or slack webhook event for "scanning finished" (SCANNIG_COMPLETED) event and expect to receieve a webhook payload that includes the image/artifact tag.

Both slack and generic http webhook receivers get an empty `"tag": ""` attribute in the received payload, additionally the `resource_url` property seems incomplete (missing the tag as well, example below).

example of event data:
```
{
    "resources": [
        {
            "digest": "sha256:e2a2fad79da79e7f37a58210689f6841f79575b4d6fe24614db17338ff0820e9",
            "tag": "",
            "resource_url": "<our container>/playground/<user>/upstream/redis:",
            "scan_overview": {
                "application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0": {
                    "report_id": "d41a5bb8-0200-4e2f-97e8-45830304dd0c",
                    "scan_status": "Success",
                    "severity": "Critical",
                    "duration": 10,
                    "summary": {
                        "total": 147,
                        "fixable": 147,
                        "summary": {
                            "Critical": 27,
                            "High": 49,
                            "Low": 10,
                            "Medium": 61
                        }
                    },
                    "start_time": "2021-02-23T17:14:31.770077Z",
                    "end_time": "2021-02-23T17:14:41.374169Z",
                    "scanner": {
                        "name": "Trivy",
                        "vendor": "Aqua Security",
                        "version": "v0.9.2"
                    },
                    "complete_percent": 100
                }
            }
        }
    ],
    "repository": {
        "name": "<user>/upstream/redis",
        "namespace": "playground",
        "repo_full_name": "playground/<user>/upstream/redis",
        "repo_type": "private"
    }
}
```

We double checked to ensure that the artifact in question indeed has a tag.

**Steps to reproduce the problem:**
1. setup webhook with either slack or http target for "scanning finished" event
2. scan an image with a tag
3. review the received webhook event data and see the tag is empty and resource_url is incomplete

**Versions:**
Please specify the versions of following systems.

- harbor version: 2.1.3

**Additional context:**

N/A but happy to provide additional information if needed.

**Comment by heww** [comment]
Duplicated with https://github.com/goharbor/harbor/issues/13464 and this issue will be fixed in v2.2.0

**Comment by dkulchinsky** [comment]
Thank you @heww and sorry for the duplicate issue, I guess I missed the other one.

but I'm a bit confused, is the `tag` going to be just omitted? or the fix will make sure the `tag` is included?
